sources/ClangSharp.PInvokeGenerator/CSharp/CSharpOutputBuilder.VisitDecl.cs

```diff
@@ -740,13 +740,18 @@ public void BeginStruct(in StructDesc desc)
             WriteLine("\"]/*' />");
         }
 
-        if (desc.LayoutAttribute is not null)
+        if (desc.LayoutAttribute is not null || desc.Layout.PackOverride is not null)
```

Comment:
Why not just have `LayoutAttribute` take this into account instead?

Reply:
That would definitely make more sense, addressed that in my latest commit.